src/core/src/any.cpp

```diff
@@ -157,6 +157,8 @@ void Any::impl_check() const {
 
 const std::type_info& Any::type_info() const {
     impl_check();
+    // `impl_check' asserts valid deference
+    // coverity[dereference:FALSE]
```

Comment:
Do we really need these comments? Is it not enough to close coverity issue as false positive ?